src/app/components/HighlightedText/text-normalization.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * Options for doing text search
+ */
+export interface NormalizerOptions {
+  /**
+   * Should we search in a case-sensitive way?
+   *
+   * (Defaults to false.)
+   */
+  caseSensitive?: boolean
+
+  /**
+   * Should we do a diacritic sensitive search?
+   *
+   * (Defaults to false.)
+   */
+  diacriticSensitive?: boolean
+}
+
+/**
+ * Groups of characters that should be considered equivalent during text searches.
+ *
+ * Currently supported languages:
+ *  - German
+ *  - Slovenian
+ *  - Croatian
+ *  - Polish
+ *  - Hungarian
+ */
+export const diacriticEquivalenceGroups: string[][] = [
```

Comment:
why we need to define all charts with diacritic? can't we use `str.normalize("NFD").replace(/[\u0300-\u036f]/g, "");`.

Reply:
Actually, yes, the method you described works great! So I could retire all the custom text normalization code. Thanks.